src/mlpack/methods/edge_boxes/feature_extraction.hpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file feature_extraction.hpp
+ * @author Nilay Jain 
+ *
+ * Feature Extraction for the edge_boxes algorithm.
+ */
+#ifndef MLPACK_METHODS_EDGE_BOXES_STRUCTURED_TREE_HPP
+#define MLPACK_METHODS_EDGE_BOXES_STRUCTURED_TREE_HPP
+//#define INF 999999.9999
+//#define EPS 1E-20
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace structured_tree {
+
+template <typename MatType = arma::mat, typename CubeType = arma::cube>
+class StructuredForests
+{
+
+ public:
+
+  static constexpr double eps = 1e-20;
+
```

Comment:
We don't use that particular parameter, also making it public isn't a good idea at all. We could just define it inside the function.